src/quantum/plugin/modifications/StatementModifaction.ts

```diff
@@ -41,8 +41,9 @@ export class StatementModification {
                     statement.setFunctionName('$fsx.r');
                     statement.setValue(resolvedFile.getID());
                 } else {
-
-                    if (core.opts.isTargetServer() || core.opts.isTargetUniveral()) {
+                    if(core.opts.isTargetNpm()) {
+                        statement.setFunctionName('require');
```

Comment:
An `npm` module uses the `require` statement to access external (vendor)  packages that are not in the bundle. If fuse-box removes all the `require`, it won't be able to bundle a package that uses other npm packages without bundeling them.

The internal `require`-s are transformed into `$fsx.s` but the ones that refer to external packages have to remain as `require`